java/src/jmri/jmrix/lenz/XNetTurnout.java

```diff
@@ -531,6 +541,46 @@ private synchronized void handleExactModeFeedback(XNetReply reply) {
         }
     }
     
+    /**
+     *  Derived from handleDirectModeFeedback.
+     *  With ONESENSOR od TWOSENSOR Mode feedback, if we see ANY valid response to our
+     *  request, we ask the command station to stop sending information
+     *  to the stationary decoder.
+     *  Only one OffMessage is sent.
+     *  <p>
+     *  No effort is made to interpret feedback when using direct mode.
+     *
+     *  @param l an {@link XNetReply} message
+     */
+    private synchronized void handleSensorModeFeedback(XNetReply l) {
+        /* If none sendOff was sent we are
```

Comment:
I'm not sure what "If _none_ sendOff was sent" means.  Is that perhaps "If _no_ sendOff was sent"?